src/App.js

```diff
@@ -142,91 +173,78 @@ const tabBarIconMap = {
-    'iPhone13,4', // iPhone 12 Pro Max
-    'iPhone14,5', // iPhone 13
-    'iPhone14,2', // iPhone 13 Pro
-    'iPhone14,3', // iPhone 13 Pro Max
-    'iPhone14,7', // iPhone 14
-    'iPhone14,8', // iPhone 14 Plus
-    'iPhone15,2', // iPhone 14 Pro
-    'iPhone15,3', // iPhone 14 Pro Max
-  ];
-
-  return workaroundDeviceIds.includes(deviceId);
+const TabNavigator = () => {
+  const Tab = createBottomTabNavigator();
+
+  return (
+    <Tab.Navigator
+      tabBarOptions={{
+        style: {
+          paddingTop: 12,
+          paddingBottom: 12,
+        },
+        tabStyle: {
+          justifyContent: 'center',
+        },
+        showIcon: true,
+        showLabel: false,
+      }}
+      screenOptions={({ route }) => ({
+        tabBarIcon: ({ focused }) => {
+          const { name } = route;
+          const iconName = tabBarIconMap[name];
+          const colorName = focused ? PRIMARY_COLOR : 'rgba(0, 0, 0, 0.5)';
+          return (<IconTabBar name={iconName} size={24} color={colorName} />);
+        },
+      })}
+    >
+      <Tab.Screen
+        name='Home'
+        component={IS_MULTI_TOKEN ? DashboardStack : MainScreen}
+      />
+      <Tab.Screen name='Send' component={SendStack} />
+      <Tab.Screen name='Receive' component={ReceiveScreen} />
+      <Tab.Screen name='Settings' component={Settings} />
+    </Tab.Navigator>
+  );
 };
 
-const TabNavigator = createBottomTabNavigator({
-  Home: (IS_MULTI_TOKEN ? DashboardStack : MainScreen),
-  Send: SendStack,
-  Receive: ReceiveScreen,
-  Settings,
-}, {
-  initialRoute: 'Home',
-  tabBarOptions: {
-    activeTintColor: PRIMARY_COLOR,
-    inactiveTintColor: 'rgba(0, 0, 0, 0.5)',
-    style: {
-      paddingTop: 12,
-      paddingBottom: 12,
-      ...(tabNavigatorMarginWorkaround() ? { marginBottom: 34 } : {})
-    },
-    tabStyle: {
-      justifyContent: 'center',
-    },
-    showIcon: true,
-    showLabel: false,
-  },
-  defaultNavigationOptions: ({ navigation }) => ({
-    tabBarIcon: ({ tintColor }) => {
-      const { routeName } = navigation.state;
-      const iconName = tabBarIconMap[routeName];
-      return <IconTabBar name={iconName} size={24} color={tintColor} />;
-    },
-  }),
-});
-
-const disableSwipeDown = () => ({
-  gesturesEnabled: false,
-});
-
-const AppStack = createStackNavigator({
-  Main: TabNavigator,
-  About,
-  Security,
-  PushNotification,
-  ChangePin,
-  ResetWallet: {
-    screen: ResetWallet,
-    // Dismissing the screen on ResetWallet would lead to a
-    // MainScreen with broken state if started from the PinScreen,
-    // so just disable the swipeDown animation
-    navigationOptions: disableSwipeDown,
-  },
-  PaymentRequestDetail,
-  RegisterToken: RegisterTokenStack,
-  ChangeToken,
-  PinScreen: {
-    screen: PinScreen,
-    navigationOptions: disableSwipeDown,
-  },
-  CreateTokenStack,
-  TokenDetail,
-  UnregisterToken,
-}, {
-  mode: 'modal',
-  headerMode: 'none',
-});
+const AppStack = () => {
```

Comment:
🚲 I think we should have a document string here like we see in InitStack.

Reply:
Done on [ae88d09](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/286/commits/ae88d09c6cdb386a0025367f7f7a6713205f4e72)